react/LazyRender.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { FunctionComponent, useRef, useState } from 'react'
+
+import { useOnView } from './hooks/useOnView'
```

Comment:
Maybe we can add this component to the app
https://github.com/vtex-apps/on-view

Reply:
Nice. I'll do it.